src/Agent.Worker/AgentPluginManager.cs

```diff
@@ -32,8 +32,9 @@ public sealed class AgentPluginManager : AgentService, IAgentPluginManager
         {
             "Agent.Plugins.Repository.CheckoutTask, Agent.Plugins",
             "Agent.Plugins.Repository.CleanupTask, Agent.Plugins",
-            "Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTask, Agent.Plugins",
-            "Agent.Plugins.PipelineArtifact.PublishPipelineArtifactTask, Agent.Plugins"
+            "Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTaskV0, Agent.Plugins",
+            "Agent.Plugins.PipelineArtifact.PublishPipelineArtifactTaskV0, Agent.Plugins",
```

Comment:
i don't think you can change those strings for V0. When agent run V0 task, the string that task.json contains is still `Agent.Plugins.PipelineArtifact.DownloadPipelineArtifactTask` and `Agent.Plugins.PipelineArtifact.PublishPipelineArtifactTask`

Reply:
oops